### DISCUSSION TITLE
filebeat module issue

### DISCUSSION THREAD

**Discussion by intsec** [Security-Onion-Solutions/securityonion#8573] [Unsupported Versions] [answered]
I have setup the filebeat module for MS Defender and I am trying to see data coming in but nothing yet.  I have recreated MS powershell fileless malware attack tutorial and I am seeing the alerts show in the Security Portal but nothing yet in the Discover logs.  What would be the best troubleshooting option for me on this?

**Comment by isaacgolding** [comment]
The logs for *Beats should be found in C:\ProgramData\Elastic\Beats .   You should start there with the basics to make sure that  filebeats is running as a service and talking to SO.

**Reply by intsec** [reply]
Filebeat is configured on the SO Manager which is not Windows.

**Comment by dougburks** [comment]
Have you checked logs in `/opt/so/log/` for additional clues?

**Reply by intsec** [reply]
Hang tight, forgot I had debugging turned on.  I am now seeing errors in filebeat logs.

**Comment by intsec** [comment, marked as answer]
So after turning off debug I was able to start seeing errors.  I was getting errors with filebeat talking to api security windows url so it made me thing there was something wrong with defender_atp module so I disabled it and now m365_defender module is working.

**Comment by intsec** [comment]
I have but I am not sure if I am looking at the right ones and what exactly I should be looking for.  Any ideas to get me started would be great.  I have looked through the filebeat logs but nothing is helping.
